shared/src/main/scala/tastyquery/reader/TreeUnpickler.scala

```diff
@@ -674,7 +740,8 @@ class TreeUnpickler(protected val reader: TastyReader, nameAtRef: NameTable) {
       case CaseDefFactory =>
         val pattern = readTerm
         val body = readTerm
-        CaseDef(pattern, reader.ifBefore(end)(readTerm, EmptyTree), body)
+        println(pattern)
+        CaseDef(pattern, reader.ifBefore(end)(readTerm, EmptyTree), body)(pattern.span.union(body.span))
```

Comment:
`CASEDEF` nodes in a tasty file are associated with their own span, so we should use that

Reply:
Fixed!